src/web/CustomScrollbar.ts

```diff
@@ -329,7 +329,24 @@ export class Scrollbar {
     }
 
     private _handleMouseDown(e: React.MouseEvent<any>) {
-        this._prevent(e);
+        const target = e.currentTarget as any;
+
+        if (this._dragging === true || !target) {
+            this._prevent(e);
+            return;
+        }
+
+        if (this._hasVertical) {
+            const eventOffsetY = e.pageY - target.getBoundingClientRect().top;
+            const halfHeight = this._verticalBar.slider!!!.offsetHeight / 2; 
+            const offsetY = (eventOffsetY - this._verticalBar.slider!!!.offsetTop - halfHeight) * this._verticalBar.slider2Scroll!!!;
+            this._viewport.scrollTop = offsetY + this._viewport.scrollTop;
+        } else if (this._hasHorizontal) {
```

Comment:
This won't work if the view has both a vertical and a horizontal scroll bar. The "else" should be removed.